WcaOnRails/config/i18n-tasks.yml.erb

```diff
@@ -85,7 +85,11 @@ search:
 #   api_key: "AbC-dEf5"
 
 ## Do not consider these keys missing:
-# ignore_missing:
+ignore_missing:
+  # Kaminari's keys, they are not prefixed by "kaminari.".
+  - 'views.pagination.*'
+  # We override devise.failure.(...) without adding the "wca" scope, but we can safely ignore the others
+  - 'devise.{mailer,passwords,registrations,sessions,shared,confirmations,unlocks}.*'
```

Comment:
Ah, right. For some reason, I was thinking that other languages were compareed again our english translations to see if anything is missing, but instead, they are compared against our codebase.